modal/_resolver.py

```diff
@@ -169,9 +163,9 @@ async def loader():
             self._local_uuid_to_future[obj.local_uuid] = cached_future
             if deduplication_key is not None:
                 self._deduplication_cache[deduplication_key] = cached_future
-
-        # TODO(elias): print original exception/trace rather than the Resolver-internal trace
-        return await cached_future
+        with suppress_tb_frames(2):
```

Comment:
I'm unsure about skipping two frames. If we may refactor the closure, then we may need to update the number of frames.

Reply:
This is a good catch, that hardcoded magic number is brittle.

Though I think it's something we could fix in a followup (since it's just about robustness to future hypothetical changes) right?

Reply:
Yea, let's fix it in a follow up.